scripts/deployment/restart_servers.sh

```diff
@@ -10,43 +10,20 @@ set -o xtrace
 
 # https://github.com/internetarchive/openlibrary/wiki/Deployment-Scratchpad
 
-dockerDown(){
-    echo "Rebooting Docker services on $SERVER with COMPOSE_FILE=$COMPOSE_FILE"
-    # WARNING! A moment of downtime 😬
-    ssh $SERVER "cd /opt/openlibrary; COMPOSE_FILE=$COMPOSE_FILE docker-compose down"
-    ssh $SERVER "docker volume rm openlibrary_ol-vendor openlibrary_ol-build openlibrary_ol-nodemodules"
-}
-
 PRODUCTION="docker-compose.yml:docker-compose.production.yml"
-STAGING="docker-compose.yml:docker-compose.staging.yml"
 
 # If no args provided then restart the services on localhost
 if [[ $@ == "" ]]; then
-    SERVERS="${HOSTNAME:-$HOST}"
+    SERVERS="${HOSTNAME}"
```

Comment:
Why cripple zsh users?

Reply:
Added in just in case; although I couldn't quite repro it with docker, but remember we had some issues of this sort.

Reply:
> since this is a bash file in the shebang, HOSTNAME would always be defined

You are correct...

% `cat ./hostname.sh`
```
#!/bin/bash

echo "HOSTNAME: $HOSTNAME"
echo "HOST: $HOST"
```
% `chmod +x hostname.sh`
% `./hostname.sh`
```
HOSTNAME: fcc-macbook.home
HOST:
```